src/routes/landing/components/Heading.js

```diff
@@ -1,22 +1,34 @@
+// @flow
+
 import React from 'react'
 
 import LocationBig from '../assets/LocationBig.svg'
-import style from './Heading.css'
 import { translate } from 'react-i18next'
 import Caption from 'modules/common/components/Caption'
-import PropTypes from 'prop-types'
+import type { TFunction } from 'react-i18next'
+import styled from 'styled-components'
 
-class Heading extends React.Component {
-  static propTypes = {
-    t: PropTypes.func.isRequired
-  }
+const Logo = styled.img`
+  display: block;
+  height: 70px;
+  margin: 0 auto;
+`
+
+const StyledCaption = styled(Caption)`
```

Comment:
We should just remove this and stick with the unstyled Caption. (Since this didn't have any effect before and I don't think we need this slight adjustment of margin here...)

Reply:
I think it does not really matter, so yes, I'll remove it.